app/dao/notifications_dao.py

```diff
@@ -849,7 +816,7 @@ def _timeout_notifications(
             Notification.status.in_(current_statuses),
             Notification.notification_type != LETTER_TYPE,
         )
-        .values({'status': new_status, 'updated_at': updated_at})
+        .values({'status': new_status, 'updated_at': updated_at, 'status_reason': status_reason})
```

Comment:
I know this wasn't added in this PR, but the `select` on 805 is unnecessary, you can just use `returning` on the update and it will give what was updated.

Reply:
I was testing this out and it may not really be worth the effort. The return of executing the update with `returning` gives a [LegacyCursorResult](https://pydoc.dev/sqlalchemy/latest/sqlalchemy.engine.cursor.LegacyCursorResult.html) and when calling `.all()` on that it provides a list of tuples (row data) with notification data instead of Notification objects. So we would need to convert those to notifications, which doesn't look like a very straightforward process at this time. I tried to do it the easy way in a list comprehension and it didn't work out. Please let me know if there's something I'm overlooking here.